bin/main/docker/mysql/init/ddl.sql

```diff
@@ -0,0 +1,128 @@
+SET CHARSET UTF8;
+SET FOREIGN_KEY_CHECKS = 0;
+drop table if exists users cascade;
```

Comment:
```suggestion
DROP TABLE IF EXISTS users CASCADE;
```
既存からですみません！予約語は大文字でお願いできますと幸いです！